docs/src/main/asciidoc/redis.adoc

```diff
@@ -0,0 +1,530 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Using Vertx Redis extension
+include::./attributes.adoc[]
+
+This guide demonstrates how your Quarkus application can use the Redis extension.
+
+[NOTE]
+====
+This extension is considered `preview`.
+API or configuration properties might change as the extension matures.
+Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing list] or as issues in our https://github.com/quarkusio/quarkus/issues[GitHub issue tracker].
+====
```

Comment:
We have a status include things for that. Have a look at the Hibernate Search Elasticsearch doc.